avm-transpiler/src/transpile.rs

```diff
@@ -968,6 +968,34 @@ fn handle_black_box_function(avm_instrs: &mut Vec<AvmInstruction>, operation: &B
                 ..Default::default()
             });
         }
+        BlackBoxOp::ToRadix {
+            input,
+            radix,
+            output,
+        } => {
+            let num_limbs = output.size;
+            let input_offset = input.0;
+            let output_offset = output.pointer.0;
+            assert!(radix <= &256u32, "Radix must be less than or equal to 256");
+
+            avm_instrs.push(AvmInstruction {
+                opcode: AvmOpcode::TORADIXLE,
+                indirect: Some(FIRST_OPERAND_INDIRECT),
```

Comment:
I think it's the second one the one that is indirect right? the first is the input field (direct) and the second one is output.pointer (indirect)

Reply:
We count from 0 XD